Astro Build fails: Cannot resolve "marked" import in feed.json.ts

### Description
The Astro build process fails with a Vite/Rollup error when trying to resolve the [marked](https://www.npmjs.com/package/marked) module import in the news feed generation.

### Steps to Reproduce
1. Navigate to the `astro` directory
2. Run `npm install`
3. Run `npm run build`
4. Build fails during the static entrypoints building phase

### Environment
- Node.js: v20.19.5
- npm: 10.8.2

### Error Message
```bash
cd astro && npm install

up to date, audited 765 packages in 874ms

309 packages are looking for funding
  run `npm fund` for details

1 high severity vulnerability

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
cd astro && npm run build

> galaxy-hub@0.0.1 build
> npm run preprocess && npm run search-index && astro build


> galaxy-hub@0.0.1 preprocess
> node src/build/preprocess.mjs

Galaxy Hub Astro v2 - Content Preprocessing
Reading from: <project-root>/content
Writing to: <project-root>/astro/src/content

Clearing existing processed content...
Found 4444 markdown files
Found 10 YAML files

Processing markdown files...
  Processed 500/4444 files...
  Processed 1000/4444 files...
  Processed 1500/4444 files...
  Processed 2000/4444 files...
  Processed 2500/4444 files...
  Processed 3000/4444 files...
  Processed 3500/4444 files...
  Processed 4000/4444 files...
Error: end of the stream or a document separator is expected at line 3, column 16:
    [getgalaxy.org](http://bitbucket.org/galaxy/gal ... 
                   ^
Error: end of the stream or a document separator is expected at line 3, column 14:
    [Get Galaxy!](http://bitbucket.org/galaxy/gal ... 
                 ^
Error: end of the stream or a document separator is expected at line 3, column 14:
    [Get Galaxy!](http://bitbucket.org/galaxy/gal ... 
                 ^
Error: end of the stream or a document separator is expected at line 3, column 14:
    [Get Galaxy!](http://bitbucket.org/galaxy/gal ... 
                 ^
Error: end of the stream or a document separator is expected at line 3, column 14:
    [Get Galaxy!](http://bitbucket.org/galaxy/gal ... 
                 ^
  Processed 4439/4444 files...
Processing YAML files...
  Processed 10/10 files...
Copying global images directory...
  Copied 1517 files from content/images/

Preprocessing complete!

Content distribution:
  articles: 2665 files
  bare-articles: 5 files
  datasets: 10 files
  events: 1364 files
  inserts: 181 files
  platforms: 224 files

Errors: 5 files failed to process

> galaxy-hub@0.0.1 search-index
> node src/build/generate-search-index.mjs

Generating search index...
  articles: 2375 entries
  events: 1361 entries
  platforms: 224 entries

Search index generated: 3960 entries
Output: <project-root>/astro/public/search-index.json
19:36:50 [vite] Re-optimizing dependencies because vite config has changed
19:36:51 [content] Syncing content
[Shiki] The language "pub" doesn't exist, falling back to "plaintext".
[Shiki] The language "pub" doesn't exist, falling back to "plaintext".
[Shiki] The language "mako" doesn't exist, falling back to "plaintext".
[Shiki] The language "Markdown" doesn't exist, falling back to "plaintext".
[Shiki] The language "Markdown" doesn't exist, falling back to "plaintext".
[Shiki] The language "hightlight" doesn't exist, falling back to "plaintext".
[Shiki] The language "hightlight" doesn't exist, falling back to "plaintext".
19:37:06 [content] Synced content
19:37:06 [types] Generated 15.65s
19:37:06 [build] output: "static"
19:37:06 [build] mode: "static"
19:37:06 [build] directory: <project-root>/astro/dist/
19:37:06 [build] Collecting build info...
19:37:06 [build] ✓ Completed in 15.69s.
19:37:06 [build] Building static entrypoints...
19:37:06 [ERROR] [vite] ✗ Build failed in 353ms
[vite]: Rollup failed to resolve import "marked" from "<project-root>/astro/src/pages/news/feed.json.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
  Stack trace:
    at viteLog (file://<project-root>/astro/node_modules/vite/dist/node/chunks/dep-D4NMHUTW.js:46374:15)
    at onLog (file://<project-root>/astro/node_modules/vite/dist/node/chunks/dep-D4NMHUTW.js:46072:7)
    at Object.logger [as onLog] (<project-root>/astro/node_modules/rollup/dist/es/shared/node-entry.js:22848:9)
    at file://<project-root>/astro/node_modules/rollup/dist/es/shared/node-entry.js:21550:26
make: *** [astro-build] Error 1
```

--- a/astro/package.json
+++ b/astro/package.json
@@ -37,6 +37,7 @@
     "glob": "^13.0.0",
     "gray-matter": "^4.0.3",
     "lucide-vue-next": "^0.555.0",
+    "marked": "^17.0.1",
     "nanostores": "^1.1.0",
     "reka-ui": "^2.6.1",
     "remark-frontmatter": "^5.0.0",

